packages/smooth_app/pubspec.yaml

```diff
@@ -10,7 +10,11 @@ dependencies:
   flutter_lints: ^2.0.1
   carousel_slider: ^4.1.1
   cupertino_icons: ^1.0.4
-  device_preview: ^1.0.0
+  # Using a fork with fixes for flutter 3 until they are applied to the package
+  device_preview:
```

Comment:
I think this change should be moved to a dedicated PR

Reply:
Leaving it in for now as it's a minor change but creating a new PR for such things in the future